Makefile

```diff
@@ -171,13 +172,14 @@ pkg_relay:
 	ls -d relay/node_modules/* | xargs -n1 -I{} cp -HR {} $(BUILD)/pkg/amon-relay/node_modules/
 	cp -PR relay/lib \
 		relay/main.js \
-		relay/package.json \
```

Comment:
Shoot, same question I asked in TritonDataCenter/sdc-config-agent#23 applies here.